docs/source/guides/localisation.rst

```diff
@@ -1,14 +1,101 @@
 Localisation
 ============
 
-Northstar adds new strings to the game which can be localised to match the language you are using on your Titanfall 2 installation.
+For your content to reach as many people as possible, it is important to have it translated in users' natural language.
+This guide will help you do that!
+
+Languages list
+--------------
+
+Languages natively supported by Titanfall2 are:
+
+* English
+* French
+* German
+* Italian
+* Japanese
+* Portuguese
+* Russian
+* Spanish
+* Traditional Chinese (``"tchinese"`` lang code)
+
+Create translation files
+------------------------
+
+Here's what a translation file looks like:
+
+.. code-block:: json
+
+    "lang"
+    {
+        "Language" "english"
+        "Tokens"
+        {
+            "MENU_LAUNCH_NORTHSTAR" "Launch Northstar"
+            "MENU_TITLE_MODS" "Mods"
+            "RELOAD_MODS" "Reload Mods"
+            "WARNING" "Warning"
+            "CORE_MOD_DISABLE_WARNING" "Disabling core mods can break your client!"
+            "DISABLE" "Disable"
+        }
+    }
+
+It begins with the ``"lang"`` instruction, contains a ``"Language"`` key indicating language of current file's translations, and 
+a ``"Token"`` key indexing all translations.
+
+.. warning ::
+    Translations files must use ``"UTF-16 LE"`` encoding.
```

Comment:
Not necessarily, this is only important if the file contains any non-ASCII characters. Not super necessary to change, but worth mentioning maybe

Reply:
I thought it was not necessary to mention since all languages but English use non-ASCII characters :smile:
Fixed anyway with 66e67e0e029bc3c22fb7b5b5c7623c210f6a2ac3.